lib/Common/Memory/LargeHeapBlock.h

```diff
@@ -29,11 +29,11 @@ struct LargeObjectHeader
 #endif
 
 public:
-    bool markOnOOMRescan;
+    bool markOnOOMRescan:1;
```

Comment:
adding static assertion for alignment

---
In reply to: [60995415](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60995415) [](ancestors = 60995415)